lib/safer_rails_console/console.rb

```diff
@@ -1,20 +1,18 @@
 module SaferRailsConsole
   module Console
     class << self
-      include SaferRailsConsole::Colors
```

Comment:
What's the reason for changing this bit? Not that I'm opposed to the change but I think this and `lib/safer_rails_console/consoles/irb.rb` should have consistent usage of this interface.